FBSnapshotTestCase/Categories/UIApplication+StrictKeyWindow.m

```diff
@@ -0,0 +1,24 @@
+//
+//  UIApplication+StrictKeyWindow.m
+//  FBSnapshotTestCase
+//
+//  Created by Brian Gerstle on 12/23/15.
+//  Copyright © 2015 Facebook. All rights reserved.
+//
+
+#import <FBSnapshotTestCase/UIApplication+StrictKeyWindow.h>
+
+@implementation UIApplication (StrictKeyWindow)
+
+- (UIWindow*)fb_strictKeyWindow {
```

Comment:
Have you guys considered using uncrustify or clang-format? 😅 While I'm at it, should I remove the comment headers (Created by Brian Gerstle etc.)?